projects/templates/projects/bundles/suites/suite.html

```diff
@@ -0,0 +1,37 @@
+{% extends "../../base.html" %}
+{% load projects_extras %}
+{% block title %}TAIPAN | Suite - {{ sd.name }}{% endblock %}
+{% block content %}
+    <h1>{{ sd.name }}</h1>
+    <div class="card">
+        <h2>Test Scenarios</h2>
+        {% with tests=sd|get_tests %}
+        {% if tests|length > 0 %}
+        <table class="list zebra">
+            <tr>
+                <th></th>
+                <th class="left-align">Description</th>
+                <th class="left-align">Required?</th>
+                <th class="right-align">Actions</th>
+            </tr>
+            {% for td in tests %}
```

Comment:
I feel like td could be confused for table data. again, probably best with `test` for this one.